lib/wlib/stl/Pair.h

```diff
@@ -0,0 +1,54 @@
+/**
+ * @file Pair.h
+ * @brief Implementation of a 2-element tuple.
+ *
+ * Created in the spirit of std::pair
+ *
+ * @author Jeff Niu
+ * @date November 2, 2017
+ * @bug No known bugs
+ */
+
+/*
+ * TODO: create and define more boolean operators
+ */
+
+#ifndef EMBEDDEDTESTS_PAIR_H
+#define EMBEDDEDTESTS_PAIR_H
+
+namespace wlp {
+
+    template<class First, class Second>
+    struct Pair {
```

Comment:
Shouldn't we make m_first and m_second private because we wanna use setters and getters?